CRM/Upgrade/Incremental/php/FiveSeventySix.php

```diff
@@ -33,7 +33,7 @@ public function upgrade_5_76_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
     $this->addTask('Add start_date to civicrm_mailing table', 'addColumn', 'civicrm_mailing', 'start_date', "timestamp NULL DEFAULT NULL COMMENT 'date on which this mailing was started.'");
     $this->addTask('Add end_date to civicrm_mailing table', 'addColumn', 'civicrm_mailing', 'end_date', "timestamp NULL DEFAULT NULL COMMENT 'date on which this mailing was completed.'");
-    $this->addTask('Add status to civicrm_mailing table', 'addColumn', 'civicrm_mailing', 'status', "varchar(12) DEFAULT NULL COMMENT 'The status of this Mailing'");
+    $this->addTask('Add status to civicrm_mailing table', 'addColumn', 'civicrm_mailing', 'status', "varchar(12) DEFAULT 'Draft' COMMENT 'The status of this Mailing'");
```

Comment:
Should the length be 12 or 32? In a stock 5.76 install it's 32, although oddly has `maxlength` 12: https://github.com/civicrm/civicrm-core/blob/f6718f2a6216ea2e7499d4603bf350e8217b111d/schema/Mailing/Mailing.entityType.php#L425-L427

Reply:
@demeritcowboy OK - I'll go with 32 for maxlength & here - seems like that makes sense